conf/defaults.ini

```diff
@@ -1123,6 +1123,10 @@ renderer_token = -
 # Concurrent render request limit affects when the /render HTTP endpoint is used. Rendering many images at the same time can overload the server,
 # which this setting can help protect against by only allowing a certain amount of concurrent requests.
 concurrent_render_request_limit = 30
+# RenderKey max duration expressed in duration (`10s`, `1m`)
+# The renderkey is used by the image renderer to access grafana (it's like an API key). The default validity is 5 minutes. This should be more than enough for most deployments.
+# Change the value only if image rendering is failing and you see `Failed to get the render key from cache` in grafana logs.
```

Comment:
```suggestion
# Determines the life time of the render key used by the image renderer to access and render Grafana.
# This setting should be expressed as a duration. Examples: 10s (seconds), 5m (minutes), 2h (hours). 
# Default is 5m. This should be more than enough for most deployments.
# Change the value only if image rendering is failing and you see `Failed to get the render key from cache` in Grafana logs.
```

Reply:
Hey @Willena,

I suggested some minor changes on the setting's description.
Could you review if those make sense for you?

If so, could you update the `conf/sample.ini` accordingly, please?
You'll see that file also contains all the available settings with the default value.

Thanks!